content/articles/managing-docker-containers-using-command-line-interface/index.md

```diff
@@ -0,0 +1,97 @@
+### Introduction
+
+With Docker, you can build, test, and deploy applications in the form of portable containers that can be used nearly anywhere. When using Docker, you might quickly get a huge number of useless items, which take up a lot of disk space and clog up the Docker command output. Unless you specifically instruct Docker too, it will not delete unneeded items like containers, images, volumes, and networks. Some of the most effective techniques to manage Docker Containers directly from the Command Line Interface are discussed in this article. We will look at some real-world examples of the most often used but very effective and efficient Docker commands that will make your Docker system organized and save up disk space by deleting unneeded Docker containers, images, volumes, and networks.
+
+### Table of contents
+
+- [Using interactive shell to run a docker container](#using-interactive-shell-to-run-a-docker-container)
+- [Remove all the dangling volumes](#remove-all-the-dangling-volumes)
+- [Removing docker containers and images](#removing-docker-containers-and-images)
+- [Using aliases](#using-aliases)
+- [Examining Docker containers](#examining-docker-containers)
+- [Conclusion](#conclusion)
+
+### Using interactive shell to run a docker container
+
+In most circumstances, an `interactive shell` reads and writes to the user's terminal. Interactive behavior is enabled when the bash command is used without any non-option arguments, unless the option is a text to read from or the shell is started to read from standard input, in which case positional parameters can be provided.
+
+Assume you have used the following command to get an Ubuntu image from Docker Hub:
+
+```bash
+sudo docker pull ubuntu
+```
+
+Now you want to use an interactive shell to execute the Ubuntu container. After downloading a Docker Ubuntu image from the official `Docker registry`, you will utilize bash on the Ubuntu OS to edit or install packages. The Docker registry is a platform that hosts images. You may do so by using the `-I` flag to execute the Docker Container in interactive mode.

```

Comment:
Please provide a link to where a reader can download the Docker Ubuntu image from its official registry.